Dockerfile.railgun

```diff
@@ -2,20 +2,12 @@
 FROM golang:1.16 as builder
 
 WORKDIR /workspace
-# Copy the Go Modules manifests
-COPY go.mod go.mod
-COPY go.sum go.sum
-# cache deps before building and copying source so that we don't need to re-download as much
-# and so that source changes don't invalidate our downloaded layer
-RUN go mod download
+COPY . .
```

Comment:
I have a small concern that the previous method only copied what was needed, this way could potentially pick up build artifacts and configuration files that aren't needed in the build image. Because we're using a multi-stage build I don't expect this will contribute to image repo bloat, but I'm curious what motivated this change?

Reply:
> what motivated this change?

When I added `manager` and `cmd/rootcmd` packages, the build container would not pick them up in the former dockerfile formulation, leading to a compilation error.

I was about to add these to the list of copied files, but stopped to think for a second and realized that maintaining a list of source directories in the Dockerfile would be tedious, with no visible gain.

> this will contribute to image repo bloat

Yes, but only for the build container which is discarded in the process of building the final image.

Reply:
After some thought, reworked this `COPY` to include only the relevant source files - 9ac045f.

This way it's better for build reproducibility.